CSS Server/Utilities/HashHelper.cs

```diff
@@ -51,11 +51,8 @@ private static byte[] CreateRandomSalt(int length)
                 randBytes = new byte[1];
             }
 
-            // Create a new RNGCryptoServiceProvider.
-            RNGCryptoServiceProvider rand = new RNGCryptoServiceProvider();
-
-            // Fill the buffer with random bytes.
-            rand.GetBytes(randBytes);
+            // Fill the buffer with a cryptographically strong random sequence of values.
+            RandomNumberGenerator.Create().GetBytes(randBytes);
```

Comment:
Maybe store the RandomNumberGenerator somewhere instead of creating it over and over again?